set.mm

```diff
@@ -507255,6 +507516,25 @@ such that every prime ideal contains a prime element (this is a
         DVCUOAUUKYRXMYOWMRHXNWPWNUUEUUFUUHWQWRUUDUUGUUHUUDUUEUUFUUDXNUUDXNUUDYG
         XMYRXNWSOAYGYRXMMWMYSXMUIAYRXMWTBCDEGXSJIYHNXAXBZVEVFZUUDXNUUMWDXEXCUUD
         XNXFOHXFOZUUHXPQUUDXNUULXDAUUNYRXMAHLXDWMXNHXGWNXHXIXJXK $.
+
+      $( Characterize elementhood to the set ` S ` of polynomials of degree
```

Comment:
"elementhood in"

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4746/commits/ee867a64dce113348e18664ad7c11b0642c11cd9